### PR TITLE
Minor addition: ports can be specified by strings (e.g., coming from com...

### PR DESCRIPTION
**Pull request by donasaur**
...mand-line args)

### CODE REVIEW

**Comment by 3rd-Eden** [comment]
Looks good, but it could use a test to ensure that this functionality is not removed in future pull requests.

File: index.js [hunk 1]
```diff
@@ -70,8 +70,9 @@ module.exports = function createServers(options, listening) {
 
     if (typeof options.http !== 'object') {
       options.http = {
-        port: typeof options.http === 'number'
-          ? options.http
+        // accept both a string and a number
+        port: (!isNaN(options.http))
```
Comment (3rd-Eden) [inline index.js]: It doesn't really require the extra parentheses here.

**Comment by donasaur** [comment]
Kk, added a test to cover the case. I was debating whether to add a test case before because I liked the symmetry displayed between the README and the test cases

File: test/create-servers-test.js [hunk 2]
```diff
@@ -103,4 +103,19 @@ test('http && https with different handlers', function (t) {
...
   });
+
+  test('only http with string type input for http port', function (t) {
+    t.plan(3);
+    createServers({
+      log: console.log,
+      http: "9876",
+      handler: fend
+    }, function (err, servers) {
+      console.dir(err);
```
Comment (3rd-Eden) [inline test/create-servers-test.js]: Is this console.dir intentional of debug leftovers?

Comment (3rd-Eden) [inline test/create-servers-test.js]: Never mind, I see that the whole test file is full of console.dirs. Ignore this comment. ;p

**Comment by 3rd-Eden** [comment]
Thanks for your contribution <3

**Pull request merged**